Update documentation for the Asset Store (#12033)
godotengine/godot-docs: community/asset_store/faq.rst

.. _doc_submitting_to_assetstore_faq:

Frequently asked questions
==========================

General store questions
-----------------------

I found a bug in the store website, where do I report it?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Please report it in the `asset store tracker repository <https://github.com/godotengine/godot-asset-store-tracker>_`.

Reviewer: ```suggestion
Please report it in the `asset store tracker repository <https://github.com/godotengine/godot-asset-store-tracker>`_.
```

Is there anywhere I can suggest new features or improvements to the store itself?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Yes! Please open an issue in the `asset store tracker repository <https://github.com/godotengine/godot-asset-store-tracker>_`.

Reviewer: ```suggestion
Yes! Please open an issue in the `asset store tracker repository <https://github.com/godotengine/godot-asset-store-tracker>`_.
```

How do I report an asset for theft/copyright infringement/malicious code?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

At the bottom of the assets page, underneath the panel on the right, there is a
"Flag This Asset" button. You **must** be logged in to use it.

> Before the Asset Store update there was an addon here I can't find anymore, was everything old removed?

Reviewer: Where is "here"?

Author: "here" being the asset library. As in "there was an asset on the asset library previously, and now that it's the asset store I can't find it" and now that I type this out I realize I should have just phrased it like this.

Author: Actually, going with this instead "I can't find an addon I download before when this was the Asset Library, was everything old removed?"

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> While the Asset Store is the successor to the Asset Library, Assets from that were

Reviewer: ```suggestion
While the Asset Store is the successor to the Asset Library, assets from there were
```

not moved over automatically. If you don't see it, the author didn't re-upload it.

> For now, the `Asset Library website <https://godotengine.org/asset-library/asset>_`

Reviewer: ```suggestion
For now, the `Asset Library website <https://godotengine.org/asset-library/asset>`_
```

is still online, and you can download it there.

Asset submission questions
--------------------------

Can I charge money for my assets?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Not currently, however that is planned and on the `roadmap <https://store.godotengine.org/roadmap/>`_.

I consider my addon in the early alpha or beta stage of development, can I upload it?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Yes. You are allowed to upload your addon even if you feel it hasn't reached 1.0
functionality. However we do ask that whatever releases you put on the store are
functional. For example, say you've only implemented 4 of your 10 features but those
4 do work (doesn't have to be bug free), that is functional. In comparison, an addon
that adds numerous buttons to the editor that don't do anything if you click on them
is non-functional.

Can I link to a Patreon, Ko-Fi, or other 3rd party site to support development?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Yes. During the submission process you can setup a donate button to link to a

Reviewer: ```suggestion
Yes. During the submission process you can set up a donate button to link to a
```

donation site. If you have multiple then you can link the others in the description.

Can I submit an addon that only works in Godot 3?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Yes.

I have an addon with a Godot 3 and Godot 4 version, do I submit it twice?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

That is what we recommend. Alternatively, you can submit it as a second version for
a single asset, however be aware that may cause confusion among users.

Is there a size limit?
~~~~~~~~~~~~~~~~~~~~~~

> Yes, a single version of an asset has a maximum size of 1GB

Reviewer: ```suggestion
Yes, a single version of an asset has a maximum size of 1GB.
```

Can I upload AI generated assets or code?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Yes, however you must disclose what is AI, or how AI was used during the

Reviewer: This needs a definition of AI, like the PR page has

.. _doc_submitting_to_assetstore_faq:

Frequently asked questions
==========================

General store questions
-----------------------

> I found a bug in the store website, where do I report it?

Reviewer 2: Maybe?
```suggestion
I found a bug in the store's website, where do I report it?
```

~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Please report it in the `asset store tracker repository <https://github.com/godotengine/godot-asset-store-tracker>`_.

Reviewer 2: Maybe?
```suggestion
Please report it in the `tracker repository for the Asset Store <https://github.com/godotengine/godot-asset-store-tracker>`_.
```

Is there anywhere I can suggest new features or improvements to the store itself?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Yes! Please open an issue in the `asset store tracker repository <https://github.com/godotengine/godot-asset-store-tracker>`_.

How do I report an asset for theft/copyright infringement/malicious code?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> At the bottom of the assets page, underneath the panel on the right, there is a

Reviewer: ```suggestion
At the bottom of the assets page, below the panel on the right, there is a
```
"Underneath" has connotations of being covered

4 do work (doesn't have to be bug free), that is functional. In comparison, an addon
that adds numerous buttons to the editor that don't do anything if you click on them
is non-functional.

Can I link to a Patreon, Ko-Fi, or other 3rd party site to support development?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Yes. During the submission process you can set up a donate button to link to a
donation site. If you have multiple then you can link the others in the description.

Can I submit an addon that only works in Godot 3?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Yes.

I have an addon with a Godot 3 and Godot 4 version, do I submit it twice?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

That is what we recommend. Alternatively, you can submit it as a second version for
a single asset, however be aware that may cause confusion among users.

Is there a size limit?
~~~~~~~~~~~~~~~~~~~~~~

Yes, a single version of an asset has a maximum size of 1GB.

Can I upload AI generated assets or code?
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

> Yes, however you must disclose what is AI generated, or how AI was used during the

Reviewer: ```suggestion
Yes, however you must disclose what is AI generated, and how AI was used during the
```

creation process when submitting the asset.

"AI" refers to any LLM/generative AI model like ChatGPT, Claude, Grok, etc., or any
other models that fulfill the same role.

If you're altering something AI generated, or using something AI generated inside of
something created manually, that still counts and you must disclose it. For example,

> if you AI generate an image, and then tweak it manually afterwards. or If you AI

Reviewer: ```suggestion
if you AI generate an image, and then tweak it manually afterwards. Or if you AI
```

> if you AI generate an image, and then tweak it manually afterwards. Or If you AI

Reviewer: ```suggestion
if you AI generate an image, and then tweak it manually afterwards. Or if you AI
```
